pkg/k8s/apis/spiderpool.spidernet.io/v2beta1/spidermultus_types.go

```diff
@@ -79,6 +79,9 @@ type SpiderMacvlanCniConfig struct {
 	// +kubebuilder:validation:Optional
 	Bond *BondConfig `json:"bond,omitempty"`
 
+	// +kubebuilder:validation:Optional
```

Comment:
there should be another filed enableRdma

Reply:
maybe we can only use one field for this. If the value is empty, which means we don't use the rdma plugin. WDYT?

Reply:
I do not think that way. Although, RdmaResourceName is a manual way. If no  enableRdma, the user has no way to tell whether turn on it or not , when it supports to automatically inject resource name

Reply:
I'm ok with if we should keep consistent with `sriov`.

> If no enableRdma, the user has no way to tell whether turn on it or not , when it supports to automatically inject resource name

ah, I think the resourceName is not to be automatically injected, configures it only manually,  the Auto-injection means configuring the resourceName into the container's resources.